Cataclysm/Rogue.lua

```diff
@@ -1291,10 +1520,15 @@ spec:RegisterAbilities( {
         startsCombat = true,
         texture = 132302,
 
-        usable = function() return combo_points.current > 0, "requires combo_points" end,
+        usable = function ()
+            if combo_points.current == 0 then return false, "requires combo_points" end
+            if ( settings.rupture_duration or 0 ) > 0 and target.time_to_die < ( settings.rupture_duration or 0 ) then return false, "target will die within " .. ( settings.rupture_duration or 0 ) .. " seconds" end
```

Comment:
Is this just copied from Retail?  I don't see a `rupture_duration` setting here.